app/controllers/ems_common.rb

```diff
@@ -832,6 +871,40 @@ def get_form_vars
     set_verify_status
   end
 
+  def set_record_creds(ems)
+    creds = {}
+    creds[:default] = {
+      :userid   => @edit[:new][:default_userid],
```

Comment:
Should I do the same for the existing set_record_vars method as well or just leave that one be?